python_modules/libraries/dagster-components/dagster_components/components/pipes_subprocess_script_collection.py

```diff
@@ -12,35 +12,24 @@
 from dagster._core.pipes.subprocess import PipesSubprocessClient
 
 from dagster_components.core.component import Component, ComponentLoadContext
-from dagster_components.resolved.core_models import AssetSpecModel, ResolvedAssetSpec
-from dagster_components.resolved.model import ResolvableModel, ResolvedFrom, Resolver
+from dagster_components.resolved.core_models import ResolvedAssetSpec
+from dagster_components.resolved.model import Resolved
 
 if TYPE_CHECKING:
     from dagster._core.definitions.definitions_class import Definitions
 
 
-class PipesSubprocessScriptModel(ResolvableModel):
-    path: str
-    assets: Sequence[AssetSpecModel]
-
-
 @dataclass
-class PipesSubprocessScript(ResolvedFrom[PipesSubprocessScriptModel]):
+class PipesSubprocessScript(Resolved):
     path: str
     assets: Sequence[ResolvedAssetSpec]
 
 
-class PipesSubprocessScriptCollectionModel(ResolvableModel):
-    scripts: Sequence[PipesSubprocessScriptModel]
-
-
 @dataclass
-class PipesSubprocessScriptCollectionComponent(
-    Component, ResolvedFrom[PipesSubprocessScriptCollectionModel]
-):
+class PipesSubprocessScriptCollectionComponent(Component, Resolved):
     """Assets that wrap Python scripts executed with Dagster's PipesSubprocessClient."""
 
-    scripts: Annotated[Sequence[PipesSubprocessScript], Resolver.from_annotation()]
+    scripts: Sequence[PipesSubprocessScript]
```

Comment:
this feels pretty good - not out of reach to make `dict[str, ResolvedAssetSpec]` work and remove the need for the intermediate obj too